lib/autoload/send_file.rb

```diff
@@ -0,0 +1,19 @@
+module SendFile
+  # Makes the given file publicly accessible.
+  #
+  # @param [String] file The path to the local file.
+  # @param [String] public_name The file's public name, default set to file's base_name.
+  # @return [String] The url to the public visible file
+  def send_file(file, public_name = File.basename(file))
+    # TODO: Support remote path and publish the file on the load balancing server.
+    downloads_dir = 'downloads/'
```

Comment:
How about use http://guides.rubyonrails.org/v4.2/configuring.html#custom-configuration to define application wide constants  ?

Like:

`config.x.public_download_folder  =  'downloads'`